renpy/config.py

```diff
@@ -1162,6 +1162,9 @@ def say_attribute_transition_callback(*args):
 # Should minimums be adjusted when x/yminimum and x/ymaximum are both floats?
 adjust_minimums = True
 
+# Should the default input caret blink ?
+input_caret_blink = 1.
```

Comment:
I think having it blink by default will result in considerable increase in frame rate on previously idle screens displaying an input, perhaps `False` ought to be the default?